### DISCUSSION TITLE
Impossible to deploy fastapi + postgres on dokku

### DISCUSSION THREAD

**Discussion by dxokal** [fastapi/fastapi#6570] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import os
from sqlalchemy import create_engine
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy.orm import sessionmaker

DEV_DB = "postgresql://postgres:postgres@localhost/bjpme"

engine = create_engine(os.getenv("DATABASE_URL", DEV_DB))
SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)

Base = declarative_base()


def get_db():
    db = SessionLocal()
    try:
        yield db
    finally:
        db.close()
```

### Description

All thing goes well on my locale machine on Debian Bulleyse. But when I processed deployment with dokku I got an error even when I set DATABASE_URL environment variable in the to the dokku app with postgres service linked to it:

 ```
return util.import_app(self.app_uri)
         File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 359, in import_app
           mod = importlib.import_module(module)
         File "/app/.heroku/python/lib/python3.6/importlib/__init__.py", line 126, in import_module
           return _bootstrap._gcd_import(name[level:], package, level)
         File "<frozen importlib._bootstrap>", line 994, in _gcd_import
         File "<frozen importlib._bootstrap>", line 971, in _find_and_load
         File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
         File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
         File "<frozen importlib._bootstrap_external>", line 678, in exec_module
         File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
         File "/app/api/main.py", line 5, in <module>
           from api.departments.main import department_route
         File "/app/api/departments/main.py", line 6, in <module>
           from api.departments import models, schemas
         File "/app/api/departments/models.py", line 4, in <module>
           from utils.database import Base
         File "/app/utils/database.py", line 8, in <module>
           engine = create_engine(DATABASE_URL)
         File "<string>", line 2, in create_engine

```

### Operating System

Linux

### Operating System Details

Debian Bulleyse

### FastAPI Version

0.68.1

### Python Version

3.9.2

### Gunicorn:

```
bind = "0.0.0.0:8080"
workers = 2
# Uvicorn's Gunicorn worker class
worker_class = "uvicorn.workers.UvicornWorker"
```
### Procfile
`web: gunicorn --worker-tmp-dir /dev/shm --config gunicorn.conf.py api.main:app
`

**Comment by Mause** [comment]
Can you paste the full error? The one you posted is missing the end

**Comment by Honda-a** [comment, marked as answer]
the problem is with your DB url
```
DEV_DB = "postgresql://postgres:postgres@localhost/bjpme"
```
change it to 
```
DEV_DB = "postgres://postgres:postgres@localhost:5432/bjpme"
```
the format is 
```
postgres://<database user><database password>@<host>:<port>/<database name>
```

**Comment by dxokal** [comment]
Thanks
